src/containers/sidebar/place/index.tsx

```diff
@@ -46,14 +46,16 @@ const Place = () => {
 
   const queryParams = useMemo(() => asPath.split('?')[1], [asPath]);
 
-  const openMenu = useCallback(() => {
-    if (!isOpen) setIsOpen(true);
-  }, [isOpen]);
+  const openLocationsModal = useCallback(() => {
+    if (!locationsModalIsOpen) setLocationsModalIsOpen(true);
+  }, [locationsModalIsOpen]);
 
   const closeMenu = useCallback(() => {
-    setIsOpen(false);
-    savePlaceOption(null);
-  }, []);
+    if (!isAnalysisAlertOpen) {
+      setLocationsModalIsOpen(false);
+      savePlaceOption(null);
+    }
+  }, [isAnalysisAlertOpen]);
 
   const handleWorldwideView = useCallback(() => {
```

Comment:
map is not flying to Worldwide

Reply:
The map is flying worldwide now, everything seems working perfectly, we are just missing the last bit (the first bullet point from the previous comment). When you open the locations modal from the Location title instead of from the sidebar, the alerts modal does not show @anamontiaga